src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Handlers/PublishContentContext.cs

```diff
@@ -2,15 +2,18 @@ namespace OrchardCore.ContentManagement.Handlers
 {
     public class PublishContentContext : ContentContextBase
     {
-        public PublishContentContext(ContentItem contentItem, ContentItem previousContentItem) : base(contentItem)
+        public PublishContentContext(ContentItem contentItem, ContentItem previousContentItem, bool? isPublishing = true) : base(contentItem)
         {
-            PublishingItem = contentItem;
+            PublishingItem = isPublishing == true ? contentItem : null;
```

Comment:
Replace all `isPublishing == true` by `isPublishing`, in all the files of this PR